test/data/ua/browser/browser-all.json

```diff
@@ -2503,6 +2503,26 @@
             "type"    : "inapp"
         }
     },
+    {
+      "desc"     : "Daum App Android",
+      "ua"       : "Mozilla/5.0 (Linux; Android 11; SM-G970N Build/RP1A.200720.012; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/83.0.4103.106 Mobile Safari/537.36 DaumApps/7.5.0 DaumDevice/mobile",
+      "expect"   : {
+        "name"   : "DaumApps",
```

Comment:
Don't forget to also change the expected name to `Daum`.

Reply:
I missed it, thanks 👋🏻